### ISSUE TITLE
Category name is required

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4742]
To reproduce:
1. Prerequisite: Logged in user
2. Click on Settings and select "Categories
3. Click on "Add category"

**Expected behaviour**
The text"Category name is required" does not appear until the user has attempted to create a category and left the field blank. 

**Actual behaviour**

The text "Category name is required" shows when Create Category page is opened.

https://github.com/ushahidi/platform/assets/8624777/dfc5ba1a-e4b8-4621-9d00-6de29479c11d

---
Created automatically from USH-781 (https://linear.app/ushahidi/issue/USH-781/category-name-is-required)

**Comment by ademolaomosanya** [comment]
hello i'm an outreachy applicant i would love to work on this task
kindly assign me

**Comment by ademolaomosanya** [comment]
@Angamanga 
i will be working on this issue 
i have the repo setup on my local machine
<img width="1280" alt="Screenshot 2024-03-05 at 20 59 01" src="https://github.com/ushahidi/platform/assets/87182204/e5d341e9-9398-431f-814f-ef38caa9a9f2">

**Comment by Appleeyes** [comment]
Please assign me to this @Angamanga

**Comment by RWUBAKWANAYO** [comment]
Hi @Angamanga ,
As an Outreachy applicant, I've completed the setup of the project locally. I'd like to work on this issue, could you please assign it to me?



![Image](https://github.com/ushahidi/platform/assets/68381641/c01ddfc6-d51a-48ea-8158-f33d425334ae)
![Image](https://github.com/ushahidi/platform/assets/68381641/6ee3e6ef-3779-4baf-9b83-44d7f5ea616a)

**Comment by Aquilaafuadajo** [comment]
@Angamanga I made a fix for this [here](https://github.com/ushahidi/platform-client-mzima/pull/818), please assign me to this issue

**Comment by ademolaomosanya** [comment]
@Angamanga I commented and had my setup running on localhost on this issue first, and I request it should be assigned to me. I also raised a [PR](#819) on this issue.
Please assign me
Thank you
